core/models.py

```diff
@@ -97,3 +97,18 @@ class Card(models.Model):
 
     def __str__(self):
         return "%s - %s (%s)" % (self.user.username, self.card_number, self.comment)
+
+
+class NfcCard(models.Model):
```

Comment:
Det har vel vært planen hele veien og bruke kortnummer på sikt. Høres ut som meg at det dere ønsker her er filtrering (vis kun kort med kortnummer f.eks.), ikke to forskjellige modeller.